src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -167,7 +167,7 @@ public static String wrap(String in, int wrapAmount) {
 
         String[] lines = in.split("\n");
         StringBuilder result = new StringBuilder();
-        addWrappedLine(result, lines[0], wrapAmount);
+        addWrappedLine(result, lines[0].replaceAll("\\s+$", ""), wrapAmount);
```

Comment:
I guess the actual question here is if we should line wrap fields when writing to the .bib-file or not? It can also be noted that there's a comment stating that this function should be "symmetric" to the parse function above, which doesn't exist... Hence, I guess that maybe one should wait a bit with sorting out this. @lenhard ?